content/projects/kinoml.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "KinoML"
+date: 2020-03-05T22:14:45+01:00
+nav: false
+draft: false
+# Uncomment sections below to enable them in your project
+external_resources:
+- name: KinoML repository
+  link: https://github.com/openkinome/kinoml
+  icon: github
+- name: KinoML documentation
+  link: https://openkinome.github.io/kinoml/
+  icon: book
+people:
+- key: jaime.rodriguez
+- key: talia.kimber
+- key: david.schaller
+- key: andrea.volkamer
+collaborators:
+- name: John D. Chodera
+  link: http://choderalab.org
+  more: MSKCC
+funding:
+- name: The Einstein Foundation & Stiftung Charité
+  link: https://www.einsteinfoundation.de/en/programmes/einstein-bih-visiting-fellow/
+  more: BIH Einstein Visiting Fellowship
+publications:
+# - proteinsplus2017
+---
+
+In this project, we aim to combine structure-enabled machine learning and alchemical free energy calculations to develop a predictive quantitative model to rapidly assess kinase inhibitor affinity and selectivity, design ligands with desired selectivity profiles, assess the impact of clinical point mutations on inhibitor binding and kinase activation, and determine the likelihood of different inhibitors to elicit mutational resistance.
+
+<!--more-->
+
+{{< xfigure src="/images/research/kinopoly.png" caption="Affinity prediction on the kinome tree." >}}
```

Comment:
@jaimergp and @t-kimber could you provide a more descriptive figure caption? What does the figure show?

Generally here could a full paragraph be added on what kinoml does/will do? Then, we might also think about changing the figure (or adding another one) that outlines the project broadly?

Reply:
you might also want to update the kinoml documentation entry page :)
https://openkinome.github.io/kinoml/

`What is KinoML?

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis facilisis consequat arcu eu tristique. Sed ut diam accumsan, iaculis leo vitae, laoreet arcu. Vivamus vel aliquam risus. Aenean magna massa, dictum ac lobortis in, porttitor at tortor. Curabitur et massa nec elit placerat fermentum. Morbi bibendum mi in felis eleifend mattis. Nullam sagittis augue at sagittis egestas. Curabitur suscipit id dolor eget feugiat. Etiam enim nunc, lobortis vitae porttitor non, suscipit eget metus. Aliquam tempor ipsum mauris, et eleifend turpis tincidunt pharetra. Mauris ex turpis, consequat sed tellus ut, dictum fringilla nunc. Donec vitae placerat orci, sed tempus elit.`

Reply:
Check last commit to see a proposal of the new fig and text. Nothing too elaborate yet. And yes, kinoml docs need some love :)

Reply:
@jaimergp  thanks I like this much better in this context, and over time would be good to elaborate on the text and add a full paragraph on the details.
For the figure just wondering, if we should use the version that was on Davids slide also this morning, which has the ligand-based part in?